### DISCUSSION TITLE
Intellij IDEA: partially accept suggestions or limit suggestion lengths

### DISCUSSION THREAD

**Discussion by the8472** [community/community#9054] [Copilot Conversations]
I often find myself rejecting copilot suggestions where a prefix of the suggestion would be useful (but still in need of correction) but the tail is nonsense and the extra work of removing the tail and then going back to correct the front is just not worth the hassle.

Additionally parsing a long suggestion takes a little more mental overhead, so being able to incrementally accept it and stop at some point also would lower the friction of scanning through the suggestion.

**Comment by baron159** [comment]
#9038 I also would very much like this

**Comment by RetriedAgain** [comment]
I am currently using the GitHub Copilot plugin for IntelliJ, and it seems like this has been added.

GitHub Copilot Version: 1.4.1.3828

![image](https://github.com/community/community/assets/57843146/2f5ed0a5-0228-4f63-baf6-f3474f879ca3)
